syntax = "proto2";
package oneflow;

enum MemoryFormat {
  kDefault = 0;
  kNCHW = 1;
  kNHWC = 2;

  MemoryFormat_Max = 3;
};
